src/it/resources/org/checkstyle/suppressionxpathfilter/modifiedcontrolvariable/InputXpathModifiedControlVariableDefault1.java

```diff
@@ -0,0 +1,9 @@
+package org.checkstyle.suppressionxpathfilter.modifiedcontrolvariable;
+
+public class InputXpathModifiedControlVariableDefault1 {
```

Comment:
Some time of This year we agreed to give meaningful names to inputs to avoid digits.
Please use InputXpathModifiedControlVariableWithFor

Reply:
done